### DISCUSSION TITLE
buddhism/2023/shunryu/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#101] [General]
# buddhism/2023/shunryu/

https://shanenull.com/buddhism/2023/shunryu/

**Comment by shane0** [comment]
# more dogen

To study the buddha way is to study the self. To study the self is to forget the self. To forget the self is to be actualized by myriad things. When actualized by myriad things, your body and mind as well as the bodies and minds of others drop away. No trace of realization remains, and this no-trace continues endlessly.
